src/core/gui/dialog/MovePageDialog.h

```diff
@@ -0,0 +1,35 @@
+/*
+ * Xournal++
+ *
+ * Move-Page dialog
+ *
+ * @author Xournal++ Team
+ * https://github.com/xournalpp/xournalpp
+ *
+ * @license GNU GPLv2 or later
+ */
+
+#pragma once
+
+#include <gtk/gtk.h>  // for GtkWindow
+
+#include "gui/GladeGui.h"  // for GladeGui
+
+class GladeSearchpath;
+
+class MovePageDialog: public GladeGui {
+public:
+    MovePageDialog(GladeSearchpath* gladeSearchPath, int currentPage, int maxPage);
```

Comment:
`size_t` for page numbers ?

Reply:
Yes, I'll change that, there is no good reason for using `int`s here